cmd/thanos/sidecar.go

```diff
@@ -45,7 +45,10 @@ import (
 func registerSidecar(app *extkingpin.App) {
 	cmd := app.Command(component.Sidecar.String(), "Sidecar for Prometheus server")
 	conf := &sidecarConfig{}
+	requestLoggingDecision := cmd.Flag("log.request.decision", "Request Logging for logging the start and end of requests. LogFinishCall is enabled by default. LogFinishCall : Logs the finish call of the requests. LogStartAndFinishCall : Logs the start and finish call of the requests. NoLogCall : Disable request logging.").Default("LogFinishCall").Enum("NoLogCall", "LogFinishCall", "LogStartAndFinishCall")
```

Comment:
This flag will be used in each command. Maybe worth to put it in some function to be used by all commands? (:

Reply:
Also would be awesome to have Log only error finish by default :thinking: Can we do this?

Reply:
Hmm , so we want to log only error and at the finish by default? That can be done :sunglasses:

Reply:
Done :+1: